onnx/defs/tensor/defs.cc

```diff
@@ -587,15 +587,15 @@ ONNX_OPERATOR_SET_SCHEMA(
           }
         }));
 
-static const char* Split_ver13_doc =
+static const char* Split_ver18_doc =
     R"DOC(Split a tensor into a list of tensors, along the specified
-'axis'. Lengths of the parts can be specified using input 'split'.
-Otherwise, the tensor is split to equal sized parts.
+'axis'. Number of equal sized outputs can be specified with 'num_outputs' 
```

Comment:
I suggest:
```
Either input 'split' or the attribute 'num_outputs' should be specified, but not both.
If the attribute 'num_outputs' is specified, then the tensor is split into equal sized parts.
If the input 'split' is specified, it indicates the sizes of each output in the split.
```

Reply:
Your version is much better. I applied it, together with `split` and `num_outputs` being optional and an error raised if both are given.